app/Jobs/ConvertActvityIntoIndependentActivity.php

```diff
@@ -62,16 +63,14 @@ public function __construct(
      * @return void
      */
     public function handle(
-        IndependentActivityRepositoryInterface $independentActivityRepository, 
+        IndependentActivityRepositoryInterface $independentActivityRepository,
         UserRepositoryInterface $userRepository
-    )
-    {
+    ) {
         try {
             $independentActivityRepository->convertIntoIndependentActivity($this->organization, $this->activity, $this->token);
             $message = "Your request to copy activity [" . $this->activity->title . "] into 
                             Independent activity has been completed and available";
-            $userId = $userRepository->parseToken($this->token);
-            $user = User::find($userId);
+            $user = User::find(Auth::id());
```

Comment:
Are sure to get "Auth::id()" ?

Reply:
Yes its ideal way of doing this, also previous function was not working